src/RepoToolset/ProjectDefaults.props

```diff
@@ -61,18 +68,18 @@
       </Choose>
     </When>
   </Choose>
-  
+
   <Import Project="|Build precondition failed: Unable to determine the language of the project. Consider adding Directory.Build.props that sets Language property to directory '$(MSBuildProjectDirectory)'.|"
           Condition="'$(Language)' == '' and '$(MSBuildProjectExtension)' != '.shproj'"/>
-  
+
   <Choose>
     <!-- C# specific settings -->
     <When Condition="'$(Language)' == 'C#'">
       <PropertyGroup>
         <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
-        <NoWarn>$(NoWarn);1591</NoWarn>
+        <NoWarn Condition="'$(NoWarn)' == ''">1701;1702;1705;1591</NoWarn>
```

Comment:
>Condition="'$(NoWarn)' == ''" [](start = 16, length = 29)

I don't think the condition is needed. These are defaults that the repo is expected to overwrite after they were defined.  The other properties in the file are also not conditional.

Reply:
Fixed.